gpu/README.md

```diff
@@ -103,6 +103,32 @@ In current Dataproc set up, we enable GPU resource isolation by initialization
 script without NVIDIA Docker, you can find more information at
 [NVIDIA Spark RAPIDS getting started guide](https://nvidia.github.io/spark-rapids/).
 
+#### cuDNN
+
+You can also install [cuDNN](https://developer.nvidia.com/CUDNN) on your
+cluster. cuDNN is used as a backend for Deep Learning frameworks, such as
+TensorFlow. This feature is currently only supported on Ubuntu-based Dataproc
```

Comment:
This is not true anymore? I.e. it supported on Debian too?

Reply:
Fixed